plugins/pomodoro-timer/manager.py

```
@@ -0,0 +1,1256 @@
+        self._pinned = want
+        request: Dict[str, Any] = {
+            "request_id": str(uuid.uuid4()),
+            "plugin_id":  self.plugin_id,
+            "action":     "start" if want else "stop",
+            "timestamp":  time.time(),
+        }
+        if want:
+            request.update({"mode": "pomodoro", "duration": None, "pinned": True})
+        try:
+            self.cache_manager.set("display_on_demand_request", request)
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)